### DISCUSSION TITLE
How to backup?

### DISCUSSION THREAD

**Discussion by minsungbr** [immense/Remotely#785] [Q&A] [answered]
Hi I have installed in synology year ago with image translucency/remotely
I believe this image is not being updated and I want to install with image immybot/remotely
Is it possible to backup all my settings and list of all computer with resident agent?

**Comment by CFox243** [comment, marked as answer]
I believe you should have a mount for "var/www/remotely"
make a backup of that create new container and move that mount over to the new container.
after that, everything should load up as is.

Docker mount:
    volumes:
      - ./var/www/remotely:/remotely-data

that dir should have
- Appsettings.json
- /logs
- /recordings
- Remotely.db
- remotely.db-shm
- remotely.db-wal

However i don't use the Synology so out side of that not 100%

**Reply by minsungbr** [reply]
Thank you very much
